src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -29,6 +29,44 @@
    :flex-direction  :row
    :justify-content :space-between})
 
+(defstyle title-text
+  {:ios     {:color          styles/color-steel
+             :font-size      13
+             :letter-spacing 1}
+   :android {:color     styles/color-gray2
+             :font-size 14}})
+
+;; NOTE(oskarth): Hacky use of defstyle to get
+;; platform specific styles in place where they belong
+(defstyle subtitle-text-augment
+  {:ios     {:uppercase? false}
+   :android {:uppercase? true}})
+
+(defstyle tag-view
+  {:margin-horizontal 2
+   :ios     {:flex-direction   :column
+             :background-color "rgb(227, 235, 250)"
```

Comment:
can you move the colors to common style ? It helps keeping the number of colors in check

Reply:
done